packages/gestalt/src/Dropdown.js

```diff
@@ -220,8 +222,18 @@ export default function Dropdown({
       role="menu"
       shouldFocus
       size="xl"
+      __dangerouslyIgnoreScrollBoundaryContainerSize={
```

Comment:
let's do something more explicit.

1. this is a boolean, https://www.30secondsofcode.org/articles/s/javascript-boolean-trap
2. __dangerouslySetMaxWidth?: "30vh"

Reply:
I believe you mean `dangerouslySetMaxHeight`.

Does this need to be `dangerous` though? What if we just give users a `maxHeight` prop and recommend against using it in most situations?

Reply:
that works too

Reply:
maxHeight works. although I wanna know about what happens if first: https://github.com/pinterest/gestalt/pull/2675#issuecomment-1457221445